Fallout2/Fallout1in2/Mapper/source/scripts/10boneyard/NICOLE.ssl

```diff
@@ -116,7 +116,8 @@ procedure talk_p_proc begin
    end
    else begin
       if ((local_var(LVAR_TALKED_TO_NICOLE) == 1) and (dude_iq < 4)) then begin
-         float_msg(self_obj, mstr(286), FLOAT_MSG_NORMAL);
+         //float_msg(self_obj, mstr(286), FLOAT_MSG_NORMAL);

```

Comment:
This floating text is a vanilla style. It should remain. No idea why you think TeamX was worried about a crash here?

Reply:
Ok, now I see why... you added the non-existing voice line here. This is what causes the crash. It needs to be reverted.

Reply:
I even went ahead to have Nicole use NMessage() instead of a float, it seemed like an oddball finding even if we understand float messages can also have voicelines. (MORPH.SSL in Et Tu uses **BMessage()** instead of a float)